survive-the-monsters.js

```diff
@@ -1,42 +1,45 @@
 import * as Phaser from "./node_modules/phaser/dist/phaser.esm.js";
 
+const GRAVITY = 200;
+const SCENE_WIDTH = 1920;
+const SCENE_HEIGHT = 1080;
+const SPRITE_SIZE = 128;
+
+const actors = {};
```

Comment:
**nitpick**: Consider using a more descriptive name for the `actors` object to clarify its purpose.